client/src/app/ui/modules/scrolling-table/components/scrolling-table/scrolling-table.component.ts

```diff
@@ -220,7 +220,9 @@ export class ScrollingTableComponent<T extends Partial<Mutable<Identifiable>>>
     private onBeforeSelectionChanged(rows: T[], isChecked?: boolean): void {
         for (const row of rows) {
             const index = row.id;
-            this._dataSourceMap[index].isSelected = isChecked || !this._dataSourceMap[index].isSelected;
+            this._dataSourceMap[index].isSelected = [true, false].includes(isChecked)
+                ? isChecked
+                : !this._dataSourceMap[index].isSelected;
```

Comment:
```suggestion
            this._dataSourceMap[index].isSelected = isChecked ?? !this._dataSourceMap[index].isSelected;
```
doesn't this also do the trick? the `[true, false].includes` looks kinda hacky